tests/arp/test_wr_arp.py

```diff
@@ -1,16 +1,15 @@
-import json
 import logging
 import pytest
 
-from tests.common.helpers.assertions import pytest_assert
 from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory                             # noqa F401
 from tests.common.fixtures.ptfhost_utils import change_mac_addresses                                # noqa F401
 from tests.common.fixtures.ptfhost_utils import remove_ip_addresses                                 # noqa F401
 from tests.common.fixtures.ptfhost_utils import skip_traffic_test                                   # noqa F401
 from tests.common.storage_backend.backend_utils import skip_test_module_over_backend_topologies     # noqa F401
 from tests.ptf_runner import ptf_runner
 from tests.common.utilities import wait_until
-
+from tests.common.arp_utils import setupFerret, teardownRouteToPtfhost, setupRouteToPtfhost, \
+    PTFRUNNER_QLEN, VXLAN_CONFIG_FILE, DEFAULT_TEST_DURATION, testWrArp
```

Comment:
Hi @yutongzhang-microsoft, this change causes the basic test test_wr_arp executed twice because the function name testWrArp starts with "test". It is executed once in the import, and again in test_wr_arp.
Could you please take a look? It should be simple to fix this by changing the name of testWrArp.

![Image](https://github.com/user-attachments/assets/52bf070c-9cb7-40b2-a71c-af3714f69257)